src/Pages/MyOrders/MyOrders.js

```diff
@@ -1 +1,140 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import MyOrdersNav from "./Components/MyOrdersNav";
+import { OrderPageImg } from "../../config";
+import OrderedProduct from "./Components/OrderedProduct";
+import { OrderedDatas } from "../../mockData";
```

Comment:
```suggestion
import MyOrdersNav from "./Components/MyOrdersNav";
import OrderedProduct from "./Components/OrderedProduct";
import { OrderPageImg } from "../../config";
import { OrderedDatas } from "../../mockData";
```
import 순서 지켜주세요. 기능 구현도 중요하지만 불필요한 데이터나 컨벤션 지키는 것 굉장히 중요합니다.

Reply:
ㅠㅠ컨벤션 놓치는 부분이 있었네요.. pr 남기기 전에 꼭 꼼꼼히 살펴보도록 하겠습니다.